src/ontology/uberon-edit.obo

```diff
@@ -9985,7 +9985,6 @@ disjoint_from: UBERON:8450002 ! excretory system
 relationship: existence_ends_during UBERON:0000066 ! fully formed stage
 relationship: fma_set_term FMA:75572
 relationship: has_part UBERON:0000991 ! gonad
-relationship: part_of UBERON:0004122 ! genitourinary system
```

Comment:
Is this being removed to capture asexual reproductive systems?

Reply:
see https://github.com/obophenotype/uberon/issues/2610#issuecomment-1216574331